public/javascript/getLeagueInfo.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * @file getLeagueInfo.js
+ * Handler for getting league match info.
+ *
+ */
+'use strict'
+ 
+var request = require('request');
+var MetaInspector = require('node-metainspector');
+var cheerio = require('cheerio');
+
+var userAPI, userThreadID, endTyping, returnString;
```

Comment:
Not too familiar with node.js, but would there be an issue with the scope of these variables being global?